lib/services/project-service.ts

```diff
@@ -102,6 +104,27 @@ export class ProjectService implements IProjectService {
 		this.$fs.createDirectory(path.join(projectDir, "platforms"));
 	}
 
+	private async ensureAppResourcesExist(projectDir: string): Promise<void> {
+		let appPath = path.join(projectDir, constants.APP_FOLDER_NAME),
```

Comment:
I agree `const` is the better approach. I'll leave it as `let` now, following the whole style of the file - where all other same-constructed paths are with `let`. I'll take care to use const in the future :)